src/main/java/nvt/kts/ticketapp/service/event/EventServiceImpl.java

```diff
@@ -136,6 +143,37 @@ private void checkAvailabilityOfLocationOnDate(Long locationSchemeId, Date date)
         }
     }
 
+    public EventDTO update(Long eventId,EventDTO eventDetails){
+
+        Event event = eventRepository.getOne(eventId);
```

Comment:
Sta ako ga ne pronadje? moras baciti neki exception.. trebalo bi da vec u repozitorijumu ima metoda findOneByid koja vvrati optional.. pogledaj kako je uradjeno to sa optionalom..

Reply:
Odradio i napravio exception